DiscordChatExporter.Core/Exporting/Partitioners/IPartitioner.cs

```diff
@@ -0,0 +1,12 @@
+﻿using DiscordChatExporter.Core.Exporting.Partitioners;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DiscordChatExporter.Core.Exporting
+{
+    public interface IPartitioner
+    {
+        public bool IsLimitReached(ExportPartitioningContext context);
```

Comment:
```suggestion
        bool IsLimitReached(ExportPartitioningContext context);
```

Adding access modifiers in interface members has a special meaning -- it denotes a default interface implementation. It has no effect here because the method doesn't have a body, but generally it's better to omit access modifiers in interfaces.